.github/workflows/deploy-standalone-plugins.yml

```diff
@@ -75,8 +87,12 @@ jobs:
         with:
           dry-run: ${{ matrix.dry-run }}
         env:
-          SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
-          SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          # TODO Once the workflow is tested, we will remove the comment and use the secret SVN access.
+          #SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
+          #SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          # TODO Once the workflow is tested, we will remove this test credential.
+          SVN_PASSWORD: SVN_PASSWORD
+          SVN_USERNAME: SVN_USERNAME
```

Comment:
This will also be reverted.

Reply:
Just curious, why not keep as-is? It is using dry-run anyway

Reply:
Regarding security, if someone tests the manual workflow and forgets to select the Debug option, the workflow will attempt to update the plugin. However, it won't release the plugin since both the WP.org and plugins.json plugin have the same version.